scripts/npm.sh

```diff
@@ -26,4 +27,10 @@ npm_config_disturl=https://atom.io/download/electron \
 npm_config_target=$ELECTRON_VERSION \
 npm_config_runtime=electron \
 HOME=$ELECTRON_GYP_HOME \
-npm $*
\ No newline at end of file
+npm $*
+
+CURRENTNPMVERSION=`npm --version`
+if [[ "$CURRENTNPMVERSION" != "$SUPPORTEDNPMVERSION" ]]; then
+	YELLOW='\033[1;33m'
+	echo -e "${YELLOW}NPM version ${CURRENTNPMVERSION} is not supported with this project. We strongly recommend to use version ${SUPPORTEDNPMVERSION}"
```

Comment:
The line coloring doesn't seem to ever end for me on Mac with this code (haven't tested on Windows yet). Is there a way of resetting the color back? I can look into this a bit, or we could just remove the coloring, the warning itself is probably good enough.